packages/lib/src/Login.wc.svelte

```diff
@@ -9,12 +9,14 @@
 
   let { loading, loggedIn, userAvatar}: LoginProps = $props();
 
+  let disabled = $state(false);
+
   const login = async (api: SesamyAPI) => {
-    loading = true;
+    disabled = true;
```

Comment:
Think it's to many spinners, and we now just disable the button after it's clicked. Also has the benefit that you can't start multiple login sessions.